invenio.cfg

```diff
@@ -510,6 +514,8 @@ APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS = [
 ]
 VOCABULARIES_NAMES_SCHEMES = {
     **DEFAULT_VOCABULARIES_NAMES_SCHEMES,
-    "cern": {"label": _("CERN"), "validator": schemes.is_cern, "datacite": "CERN"},
+    "lcds": {"label": _("CDS"), "validator": schemes.is_cern, "datacite": "CDS"},
+    "inspire": {"label": _("Inspire"), "validator": schemes.is_cern, "datacite": "INSPIRE"},
```

Comment:
shouldn't this be using [is_inspire](https://github.com/CERNDocumentServer/cds-rdm/blob/f48ace311490e1c777730c6705110d1425e0a1c6/site/cds_rdm/schemes.py#L50)?